src/fetchCoingeckoPrices.ts

```diff
@@ -0,0 +1,64 @@
+import fetch from 'isomorphic-fetch'
+import { TOKENS } from './entities/Tokens'
+
+
+const COIN_GECKO_API_KEY = process.env.COIN_GECKO_API_KEY
+
```

Comment:
Extra white spaces here in line 4 and 6. We could run prettier on commit (another task...)